sentinelsat/scripts/cli.py

```diff
@@ -137,6 +137,8 @@ def cli(user, password, geometry, start, end, uuid, name, download, sentinel, pr
 
     if geometry is not None:
         search_kwargs['area'] = geojson_to_wkt(read_geojson(geometry))
+        if api.check_query_length(search_kwargs['area']) > 200:
+            logger.warning('Query request is too large, reduce query datapoints')
```

Comment:
Please change to `'Query is too complex. Consider using a simpler geometry.'`

Reply:
changed